src/Api.js

```diff
@@ -182,7 +181,7 @@ export default class Api extends ToolApi {
   }
 
   writeCheckData(payload = {}, checkPath) {
-    const modifiedTimestamp = generateTimestamp();
+    const modifiedTimestamp = moment().toJSON();
```

Comment:
Why are we switching to moment? is moment fatster than the previous generateTimestamp? if so I would suggest using date-fns. Its better optimized and a smaller size library. Highly recommended by the Javascript community. 

For more details see: https://medium.com/@k2u4yt/momentjs-vs-date-fns-6bddc7bfa21e

https://www.npmjs.com/package/date-fns

Reply:
Ok, I see. It also looks like `date-fns` does not have a toJSON method